lua/ulx_yeet/sh_yeet.lua

```diff
@@ -163,6 +169,7 @@ local function playerDrop( ply, ent )
     ent:SetMoveType( MOVETYPE_WALK )
     ent:SetVelocity( newVelocity * 50 )
 
+    if not ent:Alive() or ent.ragdoll then return end
```

Comment:
nit: These are two separate concerns, I think it would be a good idea to make them their own lines.

Plus, saves the reader from having to figure out if this is:
```lua
if not (ent:Alive() or ent.ragdoll)
```
or
```lua
if (not ent:Alive()) or ent.ragdoll
```
```suggestion
    if not ent:Alive() then return end
    if ent.ragdoll then return end
```